spec/lib/client_summarizer_spec.rb

```diff
@@ -0,0 +1,12 @@
+describe ClientSummarizer do
+  describe "Ncr::WorkOrder" do
+    it "builds summary" do
+      wo = create(:ncr_work_order, amount: 123)
```

Comment:
you think we should create a second record to setup to make sure addition stuff is working? Also, what do you think about testing this for 18F procurement?